includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -253,7 +253,14 @@
 		<?php if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) : ?>
 			<tr>
 				<td class="label"><label for="restock_refunded_items"><?php esc_html_e( 'Restock refunded items', 'woocommerce' ); ?>:</label></td>
-				<td class="total"><input type="checkbox" id="restock_refunded_items" name="restock_refunded_items" checked="checked" /></td>
+				<?php
+				if ( apply_filter( 'woocommerce_restock_refunded_items', false ) ) {
+					$checked = 1;
+				} else {
+					$checked = 0;
+				}
+				?>
+				<td class="total"><input type="checkbox" id="restock_refunded_items" name="restock_refunded_items"<?php checked( 1, $checked, true ); ?> /></td>
```

Comment:
To mirror the old behavior it should be checked by default. Currently this will be unchecked by default. (Also there should be a space between the closing `"` and opening PHP tag).

A cleaner way of doing this whole PR might be:
```				
<td class="total"><input type="checkbox" id="restock_refunded_items" name="restock_refunded_items" <?php checked( apply_filters( 'woocommerce_restock_refunded_items', true ) ); ?> /></td>
```

Thoughts?